### PR TITLE
Feature/trade href language

### PR DESCRIPTION
**Pull request by Julian0701**
### CHECK LIST

- [Naming convention](https://github.com/CAFECA-IO/WorkGuidelines/blob/main/technology/coding-convention/naming-convention.md) verification: checked
- Coding style verification: checked
- new Library: 0
- new Class / Component:  0
- new loop: 0
- high risk: 0
- new sql: 0

### UML

- [Sequence Diagram 001 (interaction between user, frontend and backend)](https://github.com/CAFECA-IO/Documents/blob/main/TBD/TBDSD00001.md)
- [Sequence Diagram 002 (background mechanism)](https://github.com/CAFECA-IO/Documents/blob/main/TBD/TBDSD00002.md)
- [Component Diagram](https://github.com/CAFECA-IO/Documents/blob/main/TBD/TBDCP00001.md)
- [Class Diagram](https://github.com/CAFECA-IO/Documents/blob/main/TBD/TBDCD00001.md)
- [Global context document](https://www.notion.so/Global-Context-public-d93ac7c941c94fb6b173d34a7bc4e959)
- [Table of interface](https://github.com/CAFECA-IO/EventReport/blob/main/meeting/20230106_tbd_mvp.md)

### CODE REVIEW

File: src/components/crypto_summary/crypto_summary.tsx [hunk 1]
```diff
@@ -36,55 +38,13 @@ const CryptoSummary = ({
   tradingValue,
 }: ICryptoSummary) => {
   const {t}: {t: TranslateFunction} = useTranslation('common');
-
+  const {locale} = useRouter();
```
Comment (arealclimber) [inline src/components/crypto_summary/crypto_summary.tsx:41]: react-i18next 有提供現在是哪個語系的方法

```tsx
  import {useTranslation} from 'next-i18next';
  const {i18n} = useTranslation('common');
  const language = i18n.language;
  ...
```

Comment (arealclimber) [inline src/components/crypto_summary/crypto_summary.tsx:41]: next-i18next 有提供現在是哪個語系的方法

```tsx
  import {useTranslation} from 'next-i18next';
  const {i18n} = useTranslation('common');
  const language = i18n.language;
  ...
```

**Review by arealclimber — COMMENTED** [review]
提供另一種寫法，從 i18n 判斷現在是哪個語系，不用從路徑判斷

**Review by Luphia — APPROVED** [review]
- no bug found

**Pull request merged**